www/src/data/sidebars/doc-links.yaml

```diff
@@ -1,128 +1,221 @@
-      link: /docs/querying-with-graphql/
+    - title: Path Prefix
+      link: /docs/path-prefix/
+    - title: How Gatsby Works with GitHub pages
+      link: /docs/how-gatsby-works-with-github-pages/
 - title: Guides
   items:
-    - title: Adding a 404 Page
-      link: /docs/add-404-page/
-    - title: Adding Images, Fonts, and Files
-      link: /docs/adding-images-fonts-files/
-    - title: Adding Images to a WordPress Site
-      link: /docs/image-tutorial/
-    - title: Babel.js
-      link: /docs/babel/
-    - title: Browser Support
-      link: /docs/browser-support/
-    - title: Building Apps with Gatsby
+    - title: Environment*
+      link: /docs/environment/
+      items:
+      - title: Basic hardware and software requirements*
+        link: /docs/basic-hardware-software-requirements/
+      - title: Browser Support
+        link: /docs/browser-support/
+      - title: Gatsby on Macs*
+        link: /docs/gatsby-on-macs/
+      - title: Gatsby on Windows
+        link: /docs/gatsby-on-windows/
+    - title: Customization*
+      link: /docs/customization/
+      items:
+        - title: Babel.js
+          link: /docs/babel/
+        - title: Custom webpack config
+          link: /docs/add-custom-webpack-config/
+        - title: Customizing html.js
+          link: /docs/custom-html/
+        - title: Environment variables
+          link: /docs/environment-variables/
+        - title: Structure of Gatsby default starter*
+          link: /docs/gatsby-default-starter-structure/
+    - title: Content*
+      link: /docs/content/
+      items:
+        - title: Adding a 404 Page
+          link: /docs/add-404-page/
+        - title: Adding Images, Fonts, and Files
+          link: /docs/adding-images-fonts-files/
+        - title: Adding tags and categories to blog posts
+          link: /docs/adding-tags-and-categories-to-blog-posts/
+        - title: Working With images in Gatsby
+          link: /docs/working-with-images/
+    - title: GraphQL*
+      link: /docs/graphql/
+      items: 
+        - title: Introducing GraphiQL*
+          link: /docs/introducting-graphiql/
+        - title: Build a page with a GraphQL query*
+          link: /docs/build-a-page-with-graphql-query/
+        - title: Creating and modifying pages
+          link: /docs/creating-and-modifying-pages/
+        - title: Adding Markdown pages
+          link: /docs/adding-markdown-pages/
+        - title: Adding a list of Markdown blog posts
+          link: /docs/adding-a-list-of-markdown-blog-posts/
+        - title: Creating slugs for pages*
+          link: /docs/creating-slugs-for-pages/
+        - title: Programmatically create pages from data*
+          link: /docs/programmatically-create-pages-from-data/
+        - title: Querying data with StaticQuery
+          link: /docs/static-query/  
+    - title: Headless CMS*
+      link: /docs/headless-cms/
+      items:
+        - title: Managing content with Netlify CMS
```

Comment:
Can we add stubs for a few others here, especially where there's existing blog post documentation -- eg Wordpress, Drupal, Contentful, Prose, etc? This could expand a lot quite easily.

Reply:
Sure, happy to do that. Not sure how to title them. Is it too vague to say "Gatsby and Wordpress," "Gatsby and Contentful," etc? Should we just copy the titles of the blogposts instead?

Reply:
Yeah, for sub-topics, we can drop the explanatory words and just say "Netlify CMS", "WordPress", etc.

Reply:
Ok, in other PR's we'll address each sub-category one at a time, edit the main page first (e.g. Styling) and then edit the children of that header (e.g. Glamor, Styled Components, etc.) in both content and title.